tests/connected_agents/test_evidence_index.py

```
@@ -55,6 +61,23 @@ def test_connected_agents_evidence_index_is_complete_unique_and_honest():
             assert len(entry["source_commit"]) == 40
             assert entry["proof_refs"]
             assert entry["remaining_proof"] is None
+            commit = subprocess.run(
```

Ruff (0.16.2)

[error] `subprocess` call: check for execution of untrusted input

(S603)